crates/host/src/wasmbus/mod.rs

```diff
@@ -240,11 +241,19 @@ struct Handler {
     origin: WasmCloudEntity,
     // package -> target -> entity
     links: Arc<RwLock<HashMap<String, HashMap<String, WasmCloudEntity>>>>,
+
     /// The current link name to use for interface targets, overridable in actor code via set_target()
-    interface_link_name: Arc<RwLock<String>>,
-    // link name -> package -> interface -> target
+    interface_link_name: Arc<RwLock<LinkName>>,
+
+    /// Map of link names -> WIT ns & package -> WIT interface -> Target
+    ///
+    /// While a target may often be a component ID, it is not guaranteed to be one, and could be
+    /// some other identifier of where to send invocations, representing one or more lattice entities.
```

Comment:
This is a great callout to argue for some lattice target, as it may be a component ID, a routing group, or even just an opaque string 🤔